WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -127,6 +127,8 @@ class WordPressAppDelegate: UIResponder, UIApplicationDelegate {
             startObservingAppleIDCredentialRevoked()
         }
 
+        ABTest.start()
```

Comment:
Should this be called earlier, possibly before the welcome screen would get presented? Just thinking that any UI may be subject to an experiment, unless we're not going to support anonymous/logged-out experiments.

Reply:
Moved to before the screen is presented: 4a6484c4eb2c2c25b1c8f1f415f842392982b327

However, an experiment very early (like when the app is just started) is something that we'd need to discuss better. Since it's a request it can take a few ms and the first screen will probably be presented before.

Reply:
Good point! I've really had it in my head that ExPlat won't work for a lot of the login screens unless it's later in the flow.